docs/reverse_proxy.md

```diff
@@ -63,7 +63,7 @@ server {
 
     server_name matrix.example.com;
 
-    location ~* ^(\/_matrix|\/_synapse\/client) {
+    location ~ ^(/_matrix|/_synapse/client) {
```

Comment:
we may as well remove the backslashes here while we're at it - they are purely obfuscatory